api/v1/registry.proto

```diff
@@ -58,6 +60,27 @@ message VersionedApplicationSpec {
     repeated Flavor flavors = 40;
 }
 
+// for use when an installation spec contains multiple steps
```

Comment:
Can we add a bit more detail here so it is understood how this affects rendering, along the lines of:

To render the complete installation manifest, the manifests for each installation step are rendered, and then any customizations for the flavor are rendered on top. When installing, the intent is for the resources to be installed one step at a time, and any additional resources created by the flavor customization are installed at the end.